### DISCUSSION TITLE
About projects and Repos

### DISCUSSION THREAD

**Discussion by Dnis84** [community/community#15998] [Other Feature Feedback, Questions, & Ideas]
Hi, if i have a project, with three tecnologies... is bad idea think to have one upper level (organizationial) for the three repos?

**Comment by kim3-sudo** [comment]
By one "upper level," what do you mean?

If you mean an organization, then this is actually a good thing. The organizational account can have "ownership" over the three repositories, even if those repositories individually control different portions of the software that you're writing. However, if the technologies are part of "one" project, then you should ideally keep them in the same repository.

For example, let's say that you have three projects under one organization: a Python project for some data processing, a React project for some web development, and a C++ project for some internal company tooling. These are not part of the same project, but they relate to the same overarching topic, so they could be in individual repositories under one organization.

Conversely, let's say that you had three projects: a Python app, a Python + Flask app that takes some data from the first Python app, and a C++ project for some internal company tooling. The first Python app and the Python + Flask web app are part of the same project (and they probably share code), so they would probably be put in the same repository, even if your Flask templates are in HTML, your stylesheets in CSS, your scripts in JavaScript, and your processing scripts in Python. Mixing languages in one repository is okay! However, your C++ project, while topically related, should be put in a different repository, since it does something unrelated to the Python app and the Flask app.

Hopefully that makes sense.

**Reply by Dnis84** [reply]
Hi there, thanks for the answer.
In my specific case, I have an API (php code), a React application for information management (JS code, css...) and a Unity application, which will be the client side.
The API and information management do not share common code, they are in independent repositories, the same for the Unity project.
I really thought that I could do (but I don't know how to do it) the way to create a higher level of organization, which allows the repositories belonging to the same project to be grouped.

Thanks!!
![github](https://user-images.githubusercontent.com/62072507/166568870-4715859c-5776-47f8-b337-a3919f1b211f.jpg)
 :) something like this

**Comment by LeoHenry100** [comment]
It’s not a bad idea at all! Keeping an upper-level organization for your three repos can actually make things more structured. I’ve done something similar with projects [about Revo](https://revotechnologies.org/about/), and it helped streamline management across technologies.
